pkg/util/mount/mount_linux.go

```diff
@@ -240,13 +248,31 @@ func readProcMountsFrom(file io.Reader, out *[]MountPoint) (uint32, error) {
 func (mounter *SafeFormatAndMount) formatAndMount(source string, target string, fstype string, options []string) error {
 	options = append(options, "defaults")
 
+	// Run fsck on the disk to fix repairable issues
+	args := []string{"-a", source}
+	cmd := mounter.Runner.Command("fsck", args...)
```

Comment:
Well, I expected them to be in the path as well, and yet, we had to add the search anyway for it to work for everyone everywhere, so I would argue that means we should add the search for fsck as well.